packages/@yodaos/application/audio-focus.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @module @yodaos/audio-focus
+ */
+
+var id = 0
```

Comment:
Could this be duplicated when 2 same processes calls?

Reply:
Covered with test.